AuthenticationSystem/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+
+class UserProfile(User):
```

Comment:
So the user class will contains all the other fields right? like fullname, email, etc?
and do we really need phone number?
If possible can u keep the user profile class empty with only uuid or id field, so the authentication team will define the class.

Reply:
Sure